packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -59,12 +64,17 @@ const checkShouldUseMobileFeedLayout = (
 
 export const useFeedLayout = ({
   feedRelated = true,
+  originFeedName,
```

Comment:
I am not sure about this, I would much rather have it overwritten by mounting active feed context provider.

Reply:
The issue is the context doesn't exist everywhere and doesn't wrap everything.
Think it's half the issue with the router thing going on now.

I didn't want to make this a full refactor which I think someone even tried before.

Reply:
Or do you mean by using feedContext inside this hook to get this value?

Reply:
Yes, basically can we get away without adding this parameter?

Reply:
Yeah good point, let me try and refactor that part :)

Reply:
Need to check if we have access to all context from everywhere

Reply:
Not sure if I'm getting this right, but don't think it would work since the context would need to be different in the same page for different feeds.

Reply:
I've pushed these changes.
Just a bit afraid naming might be a bit complicated as we have to deal with 2 feedNames 🥲 (it's because in some places the context wouldn't exist)

@capJavert // @nensidosari let me know what you think (I do agree it's cleaner)

Reply:
Yeah definitely cleaner, but still complicated and can easily be misinterpreted. I think we moved to the feedName context for feed layout to unify them. 

What if we add a new feedName - named differently for this case to differentiate, but would still be within ActiveFeedNameContext. We would just need to add a modifier there as well for this feedName, but it would just keep us within the same context.

Reply:
Was gonna say the same thing for `ActiveFeedNameContext`, sounds better there.

Reply:
Yeah I think it makes sense and might be the best solution of all!

Reply:
Oh wait the issue lies in `ActiveFeedNameContext` always being higher then `ActiveFeedContext` so it will result in being empty always 🤣

Gosh I really hate this feed system setup we have.